programming_examples/basic/eltwise_exp/run.lit

```diff
@@ -0,0 +1,13 @@
+// (c) Copyright 2023 Advanced Micro Devices, Inc.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// REQUIRES: ryzen_ai
+//
+// RUN: xchesscc_wrapper aie2 -I %aietools/include -I %S/../../../aie_runtime_lib/AIE2 -c %S/../../../aie_kernels/aie2/bf16_exp.cc -o exp.o
+// RUN: xchesscc_wrapper aie2 -I %aietools/include -I. -c %S/../../../aie_runtime_lib/AIE2/lut_based_ops.cpp -o lut_based_ops.o
+// RUN: ar rvs kernels.a exp.o lut_based_ops.o
+// RUN: %python %S/aie2.py ipu 0 | aie-opt -cse -canonicalize -o ./aie.mlir
+// RUN: %python aiecc.py --aie-generate-cdo --aie-generate-ipu --no-compile-host --xclbin-name=aie.xclbin --ipu-insts-name=insts.txt ./aie.mlir
+// RUN: g++-13 %S/test.cpp -o test.exe -std=c++23 -Wall -I%S/../../../runtime_lib/test_lib %S/../../../runtime_lib/test_lib/test_utils.cpp %xrt_flags -lrt -lstdc++ -lboost_program_options -lboost_filesystem
+// RUN: XRT_HACK_UNSECURE_LOADING_XCLBIN=1 ./test.exe -x aie.xclbin -k MLIR_AIE -i insts.txt | FileCheck %s
```

Comment:
Let's not do this `XRT_HACK_UNSECURE_LOADING_XCLBIN=1` everywhere thing. Use `%run_on_ipu` or put config.environment['XRT_HACK_UNSECURE_LOADING_XCLBIN'] = "1" into the run.lit.py

Reply:
added to lit.cfg.py in #1253